layouts/partials/page-meta-links.html

```diff
@@ -1,5 +1,6 @@
 {{ if .Path }}
 {{ $gh_repo := ($.Param "github_repo") }}
+{{ $gh_project_repo := ($.Param "github_project_repo") }}
```

Comment:
Your PR comment suggested that this new "link" is intended to be a default generic link option for the theme (vs. all these links pointing to only GitHub), but these variables are qualified with "gh" and "github". Do you want the variables to be generic as well?